src/applications/personalization/profile360/containers/PaymentInformation.jsx

```diff
@@ -167,14 +167,11 @@ class PaymentInformation extends React.Component {
       paymentInformation,
       paymentAccount,
       shouldShowDirectDeposit,
+      directDepositIsBlocked,
     } = this.props;
 
     let content = null;
 
-    if (!shouldShowDirectDeposit) {
```

Comment:
Moved this case down below the `isLoading` check. The loading indicator was never actually being shown because `shouldShowDirectDeposit` would be `false` while things were loading (ie, waiting on the response from the `GET payment_information` endpoint.